### ISSUE TITLE
External link icon in high contrast dark theme appears as black

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4906]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

External link icon in high contrast dark theme is not visible.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS, macOS: Safari with VoiceOver, iOS/iPadOS: Safari with VoiceOver, Android: Chrome with TalkBack

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Forced colors (high contrast mode)

### How to reproduce the issue?

1. Turn on high contrast mode with dark theme (aquatic, or `forced-color: active` and `prefers-color-scheme: dark`)
2. Type `markdown` to MockBot

### What do you expect?

The external link icon should be yellow (same as `LinkText` color).

### What actually happened?

The external link icon appears to be black.

### Do you have any screenshots or recordings to repro the issue?

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/39adecd3-2b0c-459e-a677-aae37e57f19b)
